libnavui-dropin/src/main/res/layout/mapbox_partial_trip_progress.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Let's remove this file and transfer the contents directly to the layouts using it